### DISCUSSION TITLE
Custom view: how do I display a field?

### DISCUSSION THREAD

**Discussion by firecentaur** [Laravel-Backpack/community-forum#255] [Q&A (Help)] [answered]
I am very surprised that the documentation does not show how to display a field in a custom view.  there are so many fields in backpack pro, but, how on earth do we use them in a custom view?

The docs only say you can create a custom view, but no where does it show how to actually display a field in the custom view.

Please please please enlighten me!!!!

**Comment by pxpm** [comment, marked as answer]
Hello @firecentaur , We decided to not document it yet since our fields are tightly coupled with the `create/update` views, in next versions when they become components that bit will be way way more clear and easy to use. 

For now, you can include the field view with: `@include($crud->getFirstFieldView($field['type'], $field['view_namespace'] ?? false), $field)`
